test/features/support/hooks.js

```diff
@@ -0,0 +1,38 @@
+/* jshint node: true */
+
+'use strict';
+
+var myHooks = function () {
+
+  /**
+   * BeforeFeatures handler
+   *
+   * Provides access to the pageObjectsMap
+   * on browser
+   *
+   * @params {event} event The event being attached to
+   * @params {function} callback The callback function of the hook
+   */
+  this.registerHandler('BeforeFeatures', function (event, callback) {
+    browser.pageObjects = require('./pageObjectMap.js');
```

Comment:
If we renamed this to `pages` would you then suggest renaming the file to `pageMap.js` and removing any reference to `object` in that file?